packages/core/src/App/Containers/WelcomeModal/welcome-modal-cr.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import { connect } from 'Stores/connect';
+import { Modal, ThemedScrollbars } from '@deriv/components';
+import Welcome from './welcome-cr.jsx';
+
+const WelcomeModal = props => {
+    const { toggleWelcomeModal, history, toggleShouldShowMultipliersOnboarding } = props;
+    const switchPlatform = React.useCallback(
+        ({ route, should_show_multiplier } = {}) => {
+            toggleWelcomeModal({ is_visible: false, should_persist: true });
+            if (route) history.push(route);
+            if (should_show_multiplier) toggleShouldShowMultipliersOnboarding(true);
+        },
+        [toggleWelcomeModal, history, toggleShouldShowMultipliersOnboarding]
+    );
+
+    return (
+        <Modal width='760px' className='welcome welcome-cr' is_open has_close_icon={false} has_outer_content>
```

Comment:
is there a separate mobile component for this?

Reply:
no